FND/Arrangements/Communications.rdf

```diff
@@ -59,10 +59,17 @@
 	
 	<owl:Class rdf:about="&fibo-fnd-arr-com;Application">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-com;Request"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-arr-com;hasApplicationDate"/>
```

Comment:
Do we really need separate properties for hasApplicationDate and hasDateReceived?

Reply:
Those are two different things - you can submit an application ages before the lender claims to have received it if there are multiple hoops that it goes through first ... depends on the process.